presto-raptor/src/main/java/com/facebook/presto/raptor/storage/StorageManagerConfig.java

```diff
@@ -336,4 +340,17 @@ public StorageManagerConfig setOneSplitPerBucketThreshold(int oneSplitPerBucketT
         this.oneSplitPerBucketThreshold = oneSplitPerBucketThreshold;
         return this;
     }
+
+    public DateTimeZone getShardDayBoundaryTimeZone()
+    {
+        return DateTimeZone.forTimeZone(TimeZone.getTimeZone(shardDayBoundaryTimeZone));
```

Comment:
That's an interesting way of creating it. Is there a way to directly create DateTimeZone from a string?